### ISSUE TITLE
A mistake in the package summary of uk.co.caprica.vlcj.factory.discovery.provider

### ISSUE THREAD

**Issue by himawari-san** [caprica/vlcj#1083]
Hello.
I'm implementing a class of DiscoveryDirectoryProvider referring to  the [package summary](http://caprica.github.io/vlcj/javadoc/4.7.0/uk/co/caprica/vlcj/factory/discovery/provider/package-summary.html) of uk.co.caprica.vlcj.factory.discovery.provider, and then I have a question.

> Applications can easily provide their own DiscoveryDirectoryProvider implementations and register the fully-qualified classnames of those implementations in the services descriptor file located at META-INF/services/uk.co.caprica.vlcj.discovery.provider.DiscoveryDirectoryProvider 

Is "META-INF/services/uk.co.caprica.vlcj.discovery.provider.DiscoveryDirectoryProvider" a mistake for 
"META-INF/services/uk.co.caprica.vlcj.**factory**.discovery.provider.DiscoveryDirectoryProvider" ?

**Comment by caprica** [comment]
I updated it, but it won't be published until the next release - I won't make a new release immediately for such a trivial documentation fix.

**Comment by caprica** [comment]
I did find a unicorn though, someone that actually reads the Javadoc, thanks!

**Comment by himawari-san** [comment]
Thank you so much. Your Javadoc and tutorials have always helped me.
(There is also the string in the page of [DirectoryProviderDiscoveryStrategy](http://caprica.github.io/vlcj/javadoc/4.7.0/uk/co/caprica/vlcj/factory/discovery/provider/DirectoryProviderDiscoveryStrategy.html) !)

Because I've not been familiar with the ServiceLoader mechanism, I copied & used the string, and fell into a hole. But I learned many things about the ServiceLoader and Maven :)
